validator/specs/lib/validateI3dmSpec.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+var validateI3dm = require('../../lib/validateI3dm');
+
+describe('validateI3dm', function() {
+
+    it('returns true if the i3dm tile is valid, returns false if the i3dm has invalid magic', function() {
```

Comment:
Remove the "returns true if the i3dm tile is valid" part from these descriptions.